docker.py

```diff
@@ -640,6 +662,28 @@ def get_numpy_choices():
     lib_dir='/usr/lib/x86_64-linux-gnu',
 )
 
+# cuTENSOR
+
+codes['cutensor1.0.1-cuda10'] = '''
+# Ubuntu
+
+RUN if [  -n "$(cat /etc/os-release | grep Ubuntu)" ]; then \\
```

Comment:
This code is tricky and not maintainable because two independent factors (cutensor and OS version) are entangled.
I wonder the code could be refactored in a cleaner way.

Reply:
I thought of two approaches.

+having codes as a 2-level dict where the os is the first level key and the library the second.

+define a bash function to install cutensor in the main Dockerfile template and just call it from that code.

Reply:
I pushed an implementation of 1.

Reply:
Thanks. I feel the second approach is better because in 1 entanglement still exists.

Reply:
pushed a commit with the second approach.
Due to dockerfile limitations, creating shell functions can be a bit tricky.